site/views/draft/popup-game.njk

```diff
@@ -23,8 +23,8 @@ aside:
     --_font-size: calc(min(var(--_height, 300px), var(--_width, 300px)) / 3);
     position: relative;
     display: inline-block;
-    height: var(--_height, 300px);
-    width: var(--_width, 300px);
```

Comment:
But do we need the `_onWheel` handler then?

Reply:
I needed the opportunity to resize the container to check different cases with available space to position popup. The best decision that I think it up was the resizeable container. But I agree that it should be removed from this component into a mixin like `wheel-resiable-container` or something like it.